t/form-parse.t

```diff
@@ -0,0 +1,18 @@
+#!perl
+
+use strict;
+use warnings;
+
+use Test::More;
+use HTML::Form;
+
+my @warn;
+$SIG{__WARN__} = sub { push( @warn, $_[0] ) };
+
+$^W = 1;
+my @f = HTML::Form->parse( "", base => "http://localhost/", foo => 1 );
+
+is( @warn, 1 );
+like( $warn[0], qr/^Unrecognized option foo in HTML::Form/ );
```

Comment:
@haarg should we be using `Test::Warnings` here?

Reply:
@manwar I think using `Test::Warnings` would be good here. I think we're standardizing on that.